client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,272 @@
 import React from 'react';
+import SmartphoneOutlinedIcon from '@mui/icons-material/SmartphoneOutlined';
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import TimerOutlinedIcon from '@mui/icons-material/TimerOutlined';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Box, Typography, Link as MUILink } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { styled } from '@mui/material/styles';
+import scrollToTop from '../../utils/scrollToTop/scrollToTop';
 
-const Footer = () => {
-  return <div></div>;
-};
+const navigationMenu = [
+  {
+    name: 'Home',
+    path: '/',
+  },
+  {
+    name: 'Catalogue',
+    path: '/catalogue',
+  },
+  {
+    name: 'Cart',
+    path: '/cart',
+  },
+];
+
+const footerGallery = [
+  {
+    imagePath: './assets/images/footer/CARPATHIANS.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/CHERNIVTSI.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/IVANOFRANKIVSK.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIV.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/KYIVCENTER.jpg',
+    altDescription: 'recent_trips_image',
+  },
+  {
+    imagePath: './assets/images/footer/PRYKARPATTIA.png',
+    altDescription: 'recent_trips_image',
+  },
+];
+
+const FooterContainer = styled('div')({
+  color: 'white',
+  backgroundColor: 'black',
+  paddingBottom: '10px',
+});
+
+const FooterWrapper = styled('div')({
+  margin: '0 auto',
+  padding: '30px',
+  maxWidth: '1200px',
+  '@media (min-width: 768px)': {
```

Comment:
please consider the option of using the styled() with props and theme in order to utilize our theme breakpoints

```suggestion
  [theme.breakpoints.up('tablet')]: {
[styling],,,,,,,
  },
```

Reply:
solved